code/modules/clothing/neck/_neck.dm

```diff
@@ -120,6 +128,8 @@
 
 /obj/item/clothing/neck/tie/add_context(atom/source, list/context, obj/item/held_item, mob/user)
 	. = ..()
+	if(!isnull(held_item))
+		context[SCREENTIP_CONTEXT_ALT_RMB] = "Wear [alternate_worn_layer == initial(alternate_worn_layer) ? "below" : "above"] suit"
```

Comment:
I think this'd show up as "Wear below suit" when you're already wearing it below your suit, so I think you'd want to invert this.
```suggestion
		context[SCREENTIP_CONTEXT_ALT_RMB] = "Wear [alternate_worn_layer == initial(alternate_worn_layer) ? "above" : "below"] suit"
```

Reply:
Thanks for catching that, I thought I'd have caught something like that during my testing when I first started this but I guess I somehow missed it